department-of-veterans-affairs/caseflow
# Sivakesh/appeals 51904 - 508: Jaws doesn't read options on the correspondence type dropdown

Resolves [APPEALS-51904](https://jira.devops.va.gov/browse/APPEALS-51904)

# Description
Added couple of Aria attributes so that the options are read by JAWS

## Testing Plan

1. Log in as INBOUND_OPS_TEAM_ADMIN_USER at 101
2. Navigate to the Correspondence cases page.
3. Select a correspondence that will take you to the Review package page.
4. Select an option from the Correspondence type dropdown and save
5. Tab through the page while JAWS application is open until you get to the Correspondence type dropdown again.
6. Then Jaws should read Correspondence type option.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
